### PR TITLE
String interpolation without args

### PR DESCRIPTION
**Pull request by anderson-joyle**
Issue #1683

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — DISMISSED** [review]

**Comment by CarlosFigueiraMSFT** [comment]
Can you add a test for this scenario?

File: src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs [hunk 1]
```diff
@@ -785,7 +785,12 @@ public override IntermediateNode Visit(VariadicOpNode node, IRTranslatorContext
             public override IntermediateNode Visit(StrInterpNode node, IRTranslatorContext context)
             {
                 Contracts.AssertValue(node);
-                Contracts.AssertValue(context);
+                Contracts.AssertValue(context);
+
+                if (node.Children.Count == 0)
+                {
+                    return new TextLiteralNode(IRContext.NotInSource(FormulaType.String), string.Empty);
+                }
```
Comment (CarlosFigueiraMSFT) [inline src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs:793]: Can you add a test for this scenario?

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by CarlosFigueiraMSFT — APPROVED** [review]
🚢

**Pull request merged**